### PR TITLE
add raftstore.capacity configuration

### PR DESCRIPTION
**Pull request by glkappe**
be consistent：
https://docs.pingcap.com/zh/tidb/dev/hybrid-deployment-topology/#%E6%B7%B7%E5%90%88%E9%83%A8%E7%BD%B2%E7%9A%84%E5%85%B3%E9%94%AE%E5%8F%82%E6%95%B0%E4%BB%8B%E7%BB%8D

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -446,6 +446,11 @@ raftstore 相关的配置项。
 + 开启 Prevote 的开关，开启有助于减少隔离恢复后对系统造成的抖动。
 + 默认值：true
 
+### capacity
+
++ 存储容量，即允许的最大数据存储大小，如果没有设置，则使用当前磁盘容量。[同盘部署需要修改此参数](https://docs.pingcap.com/zh/tidb/dev/hybrid-deployment-topology/#%E6%B7%B7%E5%90%88%E9%83%A8%E7%BD%B2%E7%9A%84%E5%85%B3%E9%94%AE%E5%8F%82%E6%95%B0%E4%BB%8B%E7%BB%8D)
```
Comment (NingLin-P) [inline tikv-configuration-file.md:451]:
```suggestion
+ 存储容量，即允许的最大数据存储大小，如果没有设置，则使用当前磁盘容量。[同盘部署需要修改此参数](/hybrid-deployment-topology.md#混合部署的关键参数介绍)
```

**Review by NingLin-P — COMMENTED** [review]
Rest LGTM

File: tikv-configuration-file.md (hunk 1 above)
Comment (TomShawn) [reply tikv-configuration-file.md:451]:
```suggestion
+ 存储容量，即允许的最大数据存储大小。如果没有设置，则使用当前磁盘容量。如果要将多个 TiKV 实例部署在同一块物理磁盘上，需要在 TiKV 配置中添加该参数，参见[混合部署的关键参数介绍](/hybrid-deployment-topology.md#混合部署的关键参数介绍)。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -446,6 +446,11 @@ raftstore 相关的配置项。
 + 开启 Prevote 的开关，开启有助于减少隔离恢复后对系统造成的抖动。
 + 默认值：true
 
+### capacity
```
Comment (TomShawn) [inline tikv-configuration-file.md:449]:
```suggestion
### `capacity`
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@en-jin19 Could you please help create a PR to manually cherry-pick this PR to release-5.4? Thanks!

**Comment by en-jin19** [comment]
> @en-jin19 Could you please help create a PR to manually cherry-pick this PR to release-5.4? Thanks!

the changes are already updated in the [5.4 branch](https://docs.pingcap.com/zh/tidb/stable/tikv-configuration-file/#capacity-1), PTAL, thanks!

**Pull request merged**